README.adoc

```diff
@@ -242,19 +279,86 @@ initialized. With the readiness probe, you can be certain the pod will only acce
 traffic when the microservice has fully started.
 
 Similarly, navigate to `{inventory-api}/system-service` and observe that the request is successful.
+endif::[]
+
+// Cloud hosted guide instruction
+ifdef::cloud-hosted[]
+In this IBM cloud environment, you need to access the services by using the Kubernetes API.
+Run the following command to start a proxy to the Kubernetes API server:
+
+```
+kubectl proxy
+```
+{: codeblock}
+
+Open another command-line session by selecting **Terminal** > **New Terminal** from the menu of the IDE.
+Run the following commands to store the proxy path of the **system** and **inventory** services.
+```
+NAMESPACE_NAME=`bx cr namespace-list | grep sn-labs- | sed 's/ //g'`
+SYSTEM_PROXY=localhost:8001/api/v1/namespaces/$NAMESPACE_NAME/services/system-service/proxy
+INVENTORY_PROXY=localhost:8001/api/v1/namespaces/$NAMESPACE_NAME/services/inventory-service/proxy
+```
+{: codeblock}
+
+Run the following echo commands to verify the variables:
+
+```
+echo $SYSTEM_PROXY && echo $INVENTORY_PROXY
+```
+{: codeblock}
+
+
+The output appears as shown in the following example:
+
+```
+localhost:8001/api/v1/namespaces/sn-labs-yourname/services/system-service/proxy
+localhost:8001/api/v1/namespaces/sn-labs-yourname/services/inventory-service/proxy
+```
+
+Make a request to the system service to see the JVM system properties with the following **curl** command:
+```
+curl -s http://$SYSTEM_PROXY/system/properties | jq
+```
+{: codeblock}
+
+The readiness probe ensures the READY state won't be `1/1`
+until the container is available to accept requests.
+Without a readiness probe, you may notice an unsuccessful response from the server.
+This scenario can occur when the container is started,
+but the application server isn't fully initialized.
+With the readiness probe, you can be certain the pod accepts traffic only
+when the microservice is fully started.
+
+Similarly, access the inventory service and observe the successful request with the following command:
+```
+curl -s http://$INVENTORY_PROXY/inventory/systems/system-service | jq
+```
+{: codeblock}
+endif::[]
 
 == Changing the ready state of the system microservice
 
-An endpoint has been provided under the `system` microservice to set it to an unhealthy 
-state in the health check. The unhealthy state will cause the readiness probe to fail. 
-Use the `curl` command to invoke this endpoint by making a POST request to 
-`{system-api}/unhealthy` -- if `curl` is unavailable then use a tool such as 
-https://www.getpostman.com/[Postman^].
+An `unhealthy` endpoint has been provided under the `system` microservice to set it to an unhealthy 
+state. The unhealthy state will cause the readiness probe to fail.
```

Comment:
The unhealthy state will cause the readiness probe to fail.
--->
The unhealthy state causes the readiness probe to fail.